esti/adapters_test.go

```diff
@@ -1,4 +1,4 @@
-package helpers
+package esti
```

Comment:
1. Do we use all the functions defined here only in Esti?
2. I'm not sure about the new file name, it says adapter_test but there are no tests

Reply:
The helper package had adapters that include upload direct.
All the tests that used direct were removed. Except the GC test that I wasn't sure if we like to remove this case - until we will enable different way, like presign.
I've moved the code into the system test (esti) so I've kept the name and added `_test` as this code should be built only when running test.